File: 백준/Bronze/28938. Конвейер/Конвейер.cs
using System;

class Q28938
{
    static void Main(string[] args)
    {
        int n = int.Parse(Console.ReadLine());
        string[] S = Console.ReadLine().Split();
        int sum = 0;
        
        for (int i = 0; i < n; i++)
        {
            sum += int.Parse(S[i]);
        }
        
        if (sum > 0)
        {
            Console.WriteLine("Right");
        }
        else if (sum < 0)
        {
            Console.WriteLine("Left");
        }
        else
        {
            Console.WriteLine("Stay");
        }
    } 
}

File: 백준/Bronze/28938. Конвейер/Конвейер.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        
        int n = sc.nextInt();
        int sum = 0;
        
        for (int i = 0; i < n; i++) {
            sum += sc.nextInt();
        }
        
        if (sum > 0) {
            System.out.println("Right");
        } else if (sum < 0) {
            System.out.println("Left");
        } else {
            System.out.println("Stay");
        }
    } 
}